frontend/packages/knative-plugin/src/components/overview/__tests__/RevisionsOverviewList.spec.tsx

```diff
@@ -44,6 +50,26 @@ describe('RevisionsOverviewList', () => {
     ).toBe(true);
   });
 
+  it('should show Resource Link if number of revisions is more than MAX_REVISIONS', () => {
+    wrapper = shallow(
+      <RevisionsOverviewList
+        revisions={mockRevisions}
+        service={MockKnativeResources.ksservices.data[0]}
```

Comment:
can just import `knativeServiceObj` if you need just service data

Reply:
done